CODEOWNERS

```diff
@@ -43,6 +43,8 @@ extensions/filters/common/original_src @snowp @klarose
 # dynamic forward proxy
 /*/extensions/clusters/dynamic_forward_proxy @mattklein123 @alyssawilk
 /*/extensions/common/dynamic_forward_proxy @mattklein123 @alyssawilk
+# wasm common extension
```

Comment:
Nit: WebAssembly

Reply:
changed to "web assembly"  Done.

Reply:
It's [WebAssembly (abbreviated Wasm)](https://webassembly.org/), please add it to the spelling dictionary to fix the CI.